src/dbus/server/dbus_thread_object_rcp.cpp

```diff
@@ -1486,6 +1488,22 @@ otError DBusThreadObjectRcp::GetTrelInfoHandler(DBusMessageIter &aIter)
 #endif // OTBR_ENABLE_TREL
 }
 
+otError DBusThreadObjectRcp::GetMultiAilDetectedHandler(DBusMessageIter &aIter)
+{
+#if OTBR_ENABLE_MULTI_AIL
```

Comment:
Is it possible that we only keep the branch where `OTBR_ENABLE_MULTI_AIL` is enabled? 

IMHO the code is more maintainable if we have fewer build time branches. cc @bukepo